src/SFA.DAS.ApprenticeFeedback.Api/Controllers/ApprenticeFeedbackResultController.cs

```diff
@@ -56,11 +56,15 @@ public async Task<IActionResult> GetApprenticeFeedbackDetails(long ukprn)
         }
 
         [HttpGet("reviews")]
-        public async Task<IActionResult> GetApprenticeFeedbackRatingSummary()
+        public async Task<IActionResult> GetApprenticeFeedbackRatingSummary([RegularExpression("^(|All|AY\\d{4})$", ErrorMessage = "Time period should be empty, 'All', or in the format 'AYdddd'")] string timePeriod)
         {
+            if (string.IsNullOrWhiteSpace(timePeriod))
+            {
+                timePeriod = "All";
```

Comment:
There is a string "All' defined - it was used before probably best to use that again instead